src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -17,15 +20,31 @@ const propTypes = {
 
         /** Whether the user is not an admin of policyExpenseChat chat */
         isOwnPolicyExpenseChat: PropTypes.bool,
+
+        /** ID of the report */
+        reportID: PropTypes.number,
     }),
 };
 const defaultProps = {
     report: {},
 };
 
 const ReportActionItemCreated = (props) => {
+    const participants = lodashGet(props.report, 'participants', []);
+    const isChatRoom = ReportUtils.isChatRoom(props.report);
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const avatarIcons = OptionsListUtils.getAvatarSources(props.report);
+
+    function navigateToDetails() {
+        if (isChatRoom || isPolicyExpenseChat) {
+            return Navigation.navigate(ROUTES.getReportDetailsRoute(props.report.reportID));
+        }
+        if (participants.length === 1) {
+            return Navigation.navigate(ROUTES.getDetailsRoute(participants[0]));
```

Comment:
Is this really necessary?
`ROUTES.getReportParticipantsRoute(props.report.reportID)` achieves the same right?

Reply:
Yes it is necessary if we use `ROUTES.getReportParticipantsRoute(props.report.reportID)` then on clicking the avatar for a single user chat it  doesn't show the user details instead it shows this page:
<img width="374" alt="Screenshot 2022-04-19 at 9 35 10 PM" src="https://user-images.githubusercontent.com/89839952/164051151-538eabab-4959-493f-adfd-e4264d6a0c39.png">

and if the user clicks room avatar it shows members page instead of room details

Reply:
Ahh makes sense